src/main/java/lwjglwindow/LWJGLWindow.java

```diff
@@ -405,6 +411,14 @@ protected boolean tick(boolean resizing)
 		else
 			glfwSetWindowSizeLimits(window, GLFW_DONT_CARE, GLFW_DONT_CARE, GLFW_DONT_CARE, GLFW_DONT_CARE);
 
+		boolean focused = glfwGetWindowAttrib(window, GLFW_FOCUSED) == GLFW_TRUE;
+
+		if (focused != prevFocused)
+		{
+			Game.screen.onFocusChange(focused);
```

Comment:
Maybe we could set a `focusChanged` bool on the class and have `Panel`/`Screen` check that instead of calling the `onFocusChange` callback here.

Reply:
This is also an option

Reply:
Sure that works